vignettes/stan-help.Rmd

```diff
@@ -0,0 +1,403 @@
+`{epinowcast}` utilises the [`{cmdstanr}`](https://mc-stan.org/cmdstanr/) package to inferface with CmdStan, the program which excutes the models written in the Stan programming language.
+It is important to understand that CmdStan is a program that is **distinct** from but interfaced through R.
+As described in the `{epinowcast}` project [README](./index.html), you will need to install [`{cmdstanr}`](https://mc-stan.org/cmdstanr/)  R package which also has the ability to install CmdStan using an R interface. 
+Additionally, you will need to make sure that the software required by CmdStan is installed and configured on your machine.
+
+## Ensuring you have the proper toolchain {.toolchain}
+
+The Stan code that is written in the `{epinowcast}` package converted[^1] to optimised C++ code and then compiled to machine readable instructions.
+Because Stan needs a several programs to execute this compilation process such as the build tool [make](https://en.wikipedia.org/wiki/Make_(software)) and a C++ compiler, you will need to ensure that your system has the appropriate supporting software, known as a toolchain.
+The steps to install this additional software are in addition to R and are **platform specific**.
+As a reminder, these installation steps occur **outside of R**.
+The Stan team has assembled very detailed instructions for each platform:
+
+* [**Windows**](https://mc-stan.org/docs/cmdstan-guide/cmdstan-installation.html#windows)
+* [**MacOS**](https://mc-stan.org/docs/cmdstan-guide/cmdstan-installation.html#macos)
+* [**Linux**](https://mc-stan.org/docs/cmdstan-guide/cmdstan-installation.html#linux)
+
+
+After completing your platform specific toolshain installation, you can move on to R.
+
+## Now install CmdStanR and CmdStan
+
+Now you can open a session of R using your favourite IDE like Rstudio or Vscode.
+You'll need to install the CmdStanR package, the R package which allows you to interface with CmdStan through R code.
+There is a [very detailed installation guide available for CmdStanR](https://mc-stan.org/cmdstanr/articles/cmdstanr.html) which provides the authorative installation instructions.
+
+Install the CmdStanR package using the below code.
+
+```r
+# In a fresh R session run the following in your R console.
+install.packages("cmdstanr", repos = c("https://mc-stan.org/r-packages/", getOption("repos")))
+
+```
+
+To ensure that your toolchain instllation occured sucessfully, run the following code in your R terminal:
+
+```r
+check_cmdstan_toolchain()
+```
+
+This function will report back if the toolchain is available and set-up as follows:
+
+```r
+#> The C++ toolchain required for CmdStan is setup properly!
+```
+
+If you do not get this message, return to [the installation instuctions](#toolchain) and ensure that all steps were followed.
+
+Assuming you have the the toolchain installed, you can install CmdStanR.
+
+```r
+cmdstanr::install_cmdstan(cores = 2)
+```
+
+# Epinowcast modeling
```

Comment:
This is a small nit but "mode*ll*ing" is used above and "mode*l*ing" is used here